plasoscaffolder/frontend/cli_output_handler.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+"""The output file handler for click"""
+import click
+
+from plasoscaffolder.frontend import output_handler
+
+
+class OutputHandlerClick(output_handler.BaseOutputHandler):
+  """Class representing the output handler for click."""
+
+  def Confirm(self, text: str, default=True, abort=True):
+    """Ask for a confirmation, either yes or no to a question.
+
+     Args:
+       text (str): prompts the user for a confirmation, with the given test as
+           the question
+       default (bool): the default for the confirmation answer. If True the
+           default is Y(es), if False the default is N(o)
+       abort (bool): if the program should abort if the user answer to the
+           confirm prompt is no. The default is an abort.
+
+     Returns:
+        bool: False if the user entered no, True if the user entered yes
+     """
+    return click.confirm(text, abort=abort, default=default)
+
+  def PrintError(self, text: str):
+    """A echo for errors with click.
+
+    Args:
+      text (str): the text to print
+    """
+    click.secho(text, fg='red', bold=True)
+
+  def PrintInfo(self, text: str):
+    """A echo for information with click.
+
+    Args:
+      text (str): the text to print
+    """
+    click.secho(text, fg='cyan')
+
+  def PrintNewLine(self):
+    """A new line added to output."""
```

Comment:
Please fix all the docstrings to align with the log2timeline style - "Prints an empty line" here, for example. I'm going to stop mentioning this now.